test/quantization/core/test_quantized_op.py

```diff
@@ -4731,95 +4731,50 @@ def trace_handler(p):
-           X_scale=st.floats(1.2, 1.6),
-           X_zero_point=st.integers(0, 4),
-           W_scale=st.lists(st.floats(0.2, 1.6), min_size=1, max_size=2),
-           W_zero_point=st.lists(st.integers(-5, 5), min_size=1, max_size=2),
-           Y_scale=st.floats(4.2, 5.6),
-           Y_zero_point=st.integers(0, 4),
-           use_bias=st.booleans())
     @override_qengines
-    def test_qconv_transpose1d(
-            self,
-            batch_size,
-            input_channels_per_group,
-            width,
-            output_channels_per_group,
-            groups,
-            kernel,
-            stride,
-            pad,
-            o_pad,
-            dilation,
-            X_scale,
-            X_zero_point,
-            W_scale,
-            W_zero_point,
-            Y_scale,
-            Y_zero_point,
-            use_bias):
+    def test_qconv_transpose1d(self):
         if not qengine_is_qnnpack():
             return  # Currently only the QNNPACK is supported
         if qengine_is_qnnpack() and (IS_PPC or TEST_WITH_UBSAN):
             return  # QNNPACK doesn't support these
-        assume(o_pad < stride and o_pad < dilation)
-
-        input_channels = input_channels_per_group * groups
-        output_channels = output_channels_per_group * groups
-        kernels = (kernel,)
-        strides = (stride,)
-        pads = (pad,)
-        o_pads = (o_pad,)
-        dilations = (dilation,)
-
-        qconv = torch.ops.quantized.conv_transpose1d
-        qconv_prepack = torch.ops.quantized.conv_transpose1d_prepack
-        conv_op = torch.nn.ConvTranspose1d(
-            in_channels=input_channels,
-            out_channels=output_channels,
-            kernel_size=kernels,
-            stride=strides,
-            padding=pads,
-            output_padding=o_pads,
-            groups=groups,
-            dilation=dilations,
-            bias=use_bias
-        )
-
-        act_qdtypes = [torch.quint8]
-        # Only qnnpack qengine supportes qint8
-        if qengine_is_qnnpack() and torch.backends.xnnpack.enabled:
-            act_qdtypes.append(torch.qint8)
-
-        for X_qdtype in act_qdtypes:
-            if X_qdtype == torch.qint8:
-                W_zero_point = [0 for i in range(len(W_zero_point))]
-
-            X_q, W_q, bias_float = self._test_qconv_impl(
-                qconv, qconv_prepack, conv_op, batch_size,
-                input_channels_per_group, (width, ),
-                output_channels_per_group, groups, kernels, strides, pads, o_pads,
-                dilations, X_scale, X_zero_point, W_scale, W_zero_point,
-                Y_scale, Y_zero_point, use_bias, use_relu=False,
-                use_channelwise=False, use_transpose=True, input_dtype=X_qdtype, output_dtype=X_qdtype)
+        # Run N loops with random parameters
+        # itertools.product will generate too many cases thus not used
+        iters = 10
+        for _ in range(iters):
+            batch_size = random.randint(1, 3)
```

Comment:
I think we want to remove randomness in the test

Reply:
we can make some of the args fixed to reduce options we test

Reply:
OK. We use fixed parameters and `itertools.product` now.